main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Looking at this while loop, do you think it will loop 10 times or not?
```suggestion
while (loop < 9):
```